### PR TITLE
HL-194

### PR DESCRIPTION
**Pull request by brettvniekerk**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-194)

- Must be merged before [frontend pr](https://github.com/horse-link/horse.link/pull/205)
- Loads config from cache or sets it if first loaded (currently takes 4-7 seconds on my internet)

### CODE REVIEW

File: src/staticConfig.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "tokenAddresses": {
+    "USDT": "0x63B3caA357f1502389AB1F6b43b6FB118473016a",
+    "DAI": "0x6B54366642BFE522D647c77C422f1e6E11F02356"
```
Comment (pondpiu) [inline src/staticConfig.json:4]: shouldn't we get token address from contract as well?

Comment (brettvniekerk) [reply src/staticConfig.json:4]: good point - will add

File: src/index.js [hunk 2]
```diff
@@ -1320,6 +1322,161 @@ app.get("/payout", async (req, res) => {
   }
 });
 
+const getMarketOracleAddress = async (registryContract, provider) => {
+  const marketAddress = await registryContract.markets(0);
```
Comment (OptimusOpus) [inline src/index.js:1326]: Could we please leave a comment like "NOTE: this logic assumes only 1 Market Oracle contract will be used at a time" just to warn our future selfs?

**Review by OptimusOpus — APPROVED** [review]

File: src/index.js (hunk 2 above)
Comment (brettvniekerk) [reply src/index.js:1324]: good idea

**Review by pondpiu — APPROVED** [review]

**Review by OptimusOpus — APPROVED** [review]

**Pull request merged**